targets/TARGET_Samsung/TARGET_SIDK_S5JS100/device/s5js100_systemreset.c

```diff
@@ -0,0 +1,332 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2018 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/* @file  : s5js100_systemreset.c
+ * @brief : board reset source file
+ * @date  : June 2019
+ *
+ * @note  : Add chip dependent feature and more detail error status
+ *
+ */
+
+
+/****************************************************************************
+  Included Files
+ ****************************************************************************/
+#include "cmsis.h"
+#include "mbed_crash_data_offsets.h"
+#include "mbed_retarget.h"
+#include "mbed_critical.h"
+#include "mbed_error.h"
+#include "mbed_error_hist.h"
+#include "mbed_interface.h"
+#include "mbed_power_mgmt.h"
+#include "mbed_stats.h"
+#include "mbed_atomic.h"
+#include "mbed_fault_handler.h"
+
+#include "rtx_os.h"
+#include <inttypes.h>
+#include <string.h>
+#ifndef NDEBUG
+#define ERROR_REPORT(ctx, error_msg, error_filename, error_line) print_error_report(ctx, error_msg, error_filename, error_line)
+static void print_error_report(const mbed_error_ctx *ctx, const char *, const char *error_filename, int error_line);
+#else
+#define ERROR_REPORT(ctx, error_msg, error_filename, error_line) ((void) 0)
+#endif
+#if MBED_CONF_PLATFORM_CRASH_CAPTURE_ENABLED
```

Comment:
Dead if?

Reply:
Looks like it's removed now. https://github.com/ARMmbed/mbed-os/pull/12106/files#diff-25aa845838d43d1ae1c81f7378a0ef67R61-R68